executor/samples/local/logger/Makefile

```diff
@@ -23,12 +23,12 @@ curl_rest_big:
 ## GRPC
 
 run_grpc_executor:
-	${BASE}/executor --sdep seldon-model --namespace default --predictor example --file ./model_grpc.yaml --http_port 8000 --grpc_port 5000 --transport grpc
+	${BASE}/executor --sdep seldon-model --namespace default --predictor example --file ./model_grpc.yaml --port 8000
 
 run_dummy_grpc_model:
 	cd ${BASE}/../examples/models/mean_classifier && make run_grpc_local 
 
 grpc_test:
-	cd ${BASE}/proto && grpcurl -d '{"data":{"ndarray":[[1.0,2.0]]}}' -plaintext -proto ./prediction.proto  0.0.0.0:5000 seldon.protos.Seldon/Predict
+	cd ${BASE}/proto && grpcurl -d '{"data":{"ndarray":[[1.0,2.0]]}}' -plaintext -proto ./prediction.proto  0.0.0.0:8000 seldon.protos.Seldon/Predict
```

Comment:
This failed for me with
```
panic: interface conversion: interface {} is nil, not string

goroutine 76 [running]:
github.com/seldonio/seldon-core/executor/predictor.(*PredictorProcess).logPayload(0xc00009aaf0, 0xc0000e8e40, 0xa, 0xc00038eee0, 0x178a2d3, 0x7, 0x19c7800, 0xc0001cf920, 0xc00013e840, 0xc00013e840)
	/home/clive/work/seldon-core/fork-seldon-core/executor/predictor/predictor_process.go:263 +0x3ff
github.com/seldonio/seldon-core/executor/predictor.(*PredictorProcess).Predict(0xc00009aaf0, 0xc0000bcaa0, 0x19c7800, 0xc0001cf920, 0xc000110720, 0x415c4f, 0x7fdd55fac490, 0x203000)
	/home/clive/work/seldon-core/fork-seldon-core/executor/predictor/predictor_process.go:271 +0xd1
github.com/seldonio/seldon-core/executor/api/grpc/seldon.GrpcSeldonServer.Predict(0x19f36e0, 0xc00009b360, 0xc0000ee1b0, 0x19e6020, 0xc000168680, 0xc0000d4000, 0x7ffc970cfc8c, 0x7, 0x19dc640, 0xc00017f860, ...)
	/home/clive/work/seldon-core/fork-seldon-core/executor/api/grpc/seldon/server.go:37 +0x28e
github.com/seldonio/seldon-core/executor/api/grpc/seldon/proto._Seldon_Predict_Handler.func1(0x19dc640, 0xc00017f860, 0x17073c0, 0xc0001582c0, 0x17a60ba, 0x1d, 0x19dc640, 0xc00017f860)
	/home/clive/work/seldon-core/fork-seldon-core/executor/api/grpc/seldon/proto/prediction.pb.go:1538 +0x86
github.com/grpc-ecosystem/go-grpc-middleware/tracing/opentracing.UnaryServerInterceptor.func1(0x19dc640, 0xc00017f3b0, 0x17073c0, 0xc0001582c0, 0xc000110560, 0xc000110580, 0x48a1d6, 0x5ec23f4b, 0x1dfcd1fb, 0x137bb6277ab6b)
	/home/clive/go/pkg/mod/github.com/grpc-ecosystem/go-grpc-middleware@v1.1.0/tracing/opentracing/server_interceptors.go:31 +0x165
github.com/grpc-ecosystem/go-grpc-middleware.ChainUnaryServer.func1.1.1(0x19dc640, 0xc00017f3b0, 0x17073c0, 0xc0001582c0, 0x20, 0x20, 0x163a180, 0xc0001582c0)
	/home/clive/go/pkg/mod/github.com/grpc-ecosystem/go-grpc-middleware@v1.1.0/chain.go:25 +0x63
github.com/seldonio/seldon-core/executor/api/metric.(*ServerMetrics).UnaryServerInterceptor.func1(0x19dc640, 0xc00017f3b0, 0x17073c0, 0xc0001582c0, 0xc000110560, 0xc0001105c0, 0x12ad3aa, 0x163a180, 0xc0001105e0, 0xc000110560)
	/home/clive/work/seldon-core/fork-seldon-core/executor/api/metric/server.go:43 +0xa0
github.com/grpc-ecosystem/go-grpc-middleware.ChainUnaryServer.func1.1.1(0x19dc640, 0xc00017f3b0, 0x17073c0, 0xc0001582c0, 0xc000156200, 0x0, 0xc000251b18, 0x40e488)
	/home/clive/go/pkg/mod/github.com/grpc-ecosystem/go-grpc-middleware@v1.1.0/chain.go:25 +0x63
github.com/grpc-ecosystem/go-grpc-middleware.ChainUnaryServer.func1(0x19dc640, 0xc00017f3b0, 0x17073c0, 0xc0001582c0, 0xc000110560, 0xc000110580, 0xc00041bb88, 0x4932e8, 0x166efc0, 0xc00017f3b0)
	/home/clive/go/pkg/mod/github.com/grpc-ecosystem/go-grpc-middleware@v1.1.0/chain.go:34 +0xd5
github.com/seldonio/seldon-core/executor/api/grpc/seldon/proto._Seldon_Predict_Handler(0x158e480, 0xc000158400, 0x19dc640, 0xc00017f3b0, 0xc00010e6c0, 0xc000256900, 0x19dc640, 0xc00017f3b0, 0xc0003a4a40, 0x1e)
	/home/clive/work/seldon-core/fork-seldon-core/executor/api/grpc/seldon/proto/prediction.pb.go:1540 +0x14b
google.golang.org/grpc.(*Server).processUnaryRPC(0xc000120b60, 0x19ef320, 0xc000298000, 0xc000156200, 0xc000256a20, 0x25fe620, 0x0, 0x0, 0x0)
	/home/clive/go/pkg/mod/google.golang.org/grpc@v1.28.0/server.go:1082 +0x50a
google.golang.org/grpc.(*Server).handleStream(0xc000120b60, 0x19ef320, 0xc000298000, 0xc000156200, 0x0)
	/home/clive/go/pkg/mod/google.golang.org/grpc@v1.28.0/server.go:1405 +0xcc9
google.golang.org/grpc.(*Server).serveStreams.func1.1(0xc0000446a0, 0xc000120b60, 0x19ef320, 0xc000298000, 0xc000156200)
	/home/clive/go/pkg/mod/google.golang.org/grpc@v1.28.0/server.go:746 +0xa1
created by google.golang.org/grpc.(*Server).serveStreams.func1
	/home/clive/go/pkg/mod/google.golang.org/grpc@v1.28.0/server.go:744 +0xa1
Makefile:26: recipe for target 'run_grpc_executor' failed
make: *** [run_grpc_executor] Error 2
```

Reply:
This is the line failing:
https://github.com/SeldonIO/seldon-core/blob/21a0cff9dcb60ba1ea4d6c87ec90bdecb27a646b/executor/predictor/predictor_process.go#L263

Which has been raised in a separate issue #1763 which will be fixed once PR #1790 is merged.